src/utilcode/ccomprc.cpp

```diff
@@ -10,7 +10,10 @@
 #include "ndpversion.h"
 
 #include "../dlls/mscorrc/resource.h"
-#include "../dlls/mscorrc/resourcestring.h"
+#ifdef PLATFORM_UNIX
+#include "resourcestring.h"
+DECLARE_STRING_RESOURCE_TABLE(STRING_RESOURCE_NAME);
```

Comment:
What I meant is that you can define the macro in this file instead of passing it from the build scripts. It would improve readability of the source since you would not have to hunt for what the name actually is and where it is defined.